FacebookAuth/FirebaseFacebookAuthUI.xcodeproj/project.pbxproj

```diff
@@ -153,12 +153,20 @@
 /* End PBXFrameworksBuildPhase section */
 
 /* Begin PBXGroup section */
+		7DC88936A359F0A20973DC4C /* Pods */ = {
```

Comment:
remove?

Reply:
These aren't removed by `pod deintegrate` for some reason. They're not harmful enough to warrant manually removing all of them.